MapboxVision/README.md

```diff
@@ -1 +0,0 @@
-#MapboxVision
```

Comment:
This file is needed for `jazzy` in order to generate an empty starting page instead of a default one.

Reply:
Oh, didn't think about that. Thank you. I've added a comment to the file to avoid removing in the future.